psse/psse-model/src/main/java/com/powsybl/psse/model/data/RawXData35.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.psse.model.data;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.io.ByteStreams;
+import com.powsybl.commons.datasource.ReadOnlyDataSource;
+import com.powsybl.psse.model.PsseCaseIdentification;
+import com.powsybl.psse.model.PsseRawModel;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class RawXData35 extends RawXDataCommon {
+
+    @Override
+    public PsseRawModel read(ReadOnlyDataSource dataSource, String ext, Context context) throws IOException {
+        String jsonFile = new String(ByteStreams.toByteArray(dataSource.newInputStream(null, ext)), StandardCharsets.UTF_8);
```

Comment:
The name of this variable is not ok: it's more a content than a file. I wonder if it's a good idea to put the content in a string and if it's possible to read the input stream directly. Moreover, the input stream should be closed.

Reply:
Agree. Changed the name of the variable. Used try-with-resources statement.

We were already investigating how to obtain json content without having to read the entire file. Current approach is valid only for the small test files we have now, but will fix it in separate PR, if you accept it.

Reply:
I change the code to pass the stream directly to the `ObjectMapper().readTree()` method instead of storing the content in a String. I'm pretty sure it doesn't change anything but it should be more efficient because the file won't be read twice (one time to fill the string, and one time by the parser).